src/main.rs

```diff
@@ -79,23 +81,66 @@ fn get_args(current_dir: &OsStr) -> clap::ArgMatches {
                 .long("no-color")
                 .help("Turns off the colored output"),
         )
-        .arg(
-            Arg::with_name("fix")
-                .short("f")
-                .long("fix")
-                .help("Automatically fixes warnings"),
-        )
-        .arg(
-            Arg::with_name("no-backup")
-                .long("no-backup")
-                .help("Prevents .env files from being backed up when modified by -f/--fix"),
-        )
         .arg(
             Arg::with_name("quiet")
                 .short("q")
                 .long("quiet")
                 .help("Doesn't display additional information"),
         )
+        .subcommand(SubCommand::with_name("checks").about("Shows list of available checks"))
+        .subcommand(
+            SubCommand::with_name("fix")
+                .alias("f")
+                .about("Automatically fixes warnings")
+                .arg(
+                    Arg::with_name("input")
+                        .help("files or paths")
+                        .index(1)
+                        .default_value_os(current_dir)
+                        .required(true)
+                        .multiple(true),
+                )
+                .arg(
+                    Arg::with_name("exclude")
```

Comment:
How do you look at make an array with common `check` and `fixed` arguments?
Something like a function `get_common_arguments()` ? 🤔 
As I see it is true for: `input`, `exclude`, `skip`, `recursive`, `no-color`, `quiet`

Reply:
I was just about to do it 😃